en/deployment/articles/03-installing-a-search-engine/02-elasticsearch/11-upgrading-to-elasticsearch-73.markdown

```diff
@@ -79,6 +79,7 @@ To blacklist Elasticsearch 6,
         "com.liferay.portal.search.elasticsearch6.impl", \
         "com.liferay.portal.search.elasticsearch6.spi", \
         "com.liferay.portal.search.elasticsearch6.xpack.security.impl", \
+        "Liferay Connector to X-Pack Security [Elastic Stack 6.x] - Impl", \
```

Comment:
@lipusz i think i should get rid of the  line above this right? it probably represents what i used for testing, if i was building the connector from src.

Reply:
It doesn't hurt to keep it.